test/facade_converter_test.ts

```diff
@@ -94,32 +94,46 @@ describe('type based translation', () => {
     it('translates array operations to dartisms', () => {
       expectWithTypes('var x: Array<number> = []; x.push(1); x.pop();')
           .to.equal(' List < num > x = [ ] ; x . add ( 1 ) ; x . removeLast ( ) ;');
+
```

Comment:
I'll remove the newlines from here (indeed, the idea was to group related tests which explain why there is no newlines between `unshift` and `unshift`)